r_packages.csv

```diff
@@ -0,0 +1,23 @@
+name,version,sha256
+BH,1.72.0-3,888ec1a3316bb69e1ba749b08ba7e0903ebc4742e3a185de8d148c13cddac8ab
+R6,2.4.1,26b0fd64827655c28c903f7ff623e839447387f3ad9b04939a02f41ac82faa3e
+Rcpp,1.0.1,e76de03e5db5edc760ba7372b688d3a2f34854ad2b7dcb53b31556a46dafb904
+assertthat,0.2.1,85cf7fcc4753a8c86da9a6f454e46c2a58ffc70c4f47cac4d3e3bcefda2a9e9f
+cli,1.1.0,4fc00fcdf4fdbdf9b5792faee8c7cf1ed5c4f45b1221d961332cda82dbe60d0a
+clipr,0.7.0,03a4e4b72ec63bd08b53fe62673ffc19a004cc846957a335be2b30d046b8c2e2
+crayon,1.3.4,fc6e9bf990e9532c4fcf1a3d2ce22d8cf12d25a95e4779adfa17713ed836fa68
+digest,0.6.24,04985b4179b69b4d5e4ad3491a3d79af5ff52991f3c00f86fa2ba98d1fabba07
+ellipsis,0.3.1,4f8a15158dfc27cdc0f7554c7a61e92b02e4d70bfc3d968f01a99da2189b75db
+fansi,0.4.1,3c69eec803a3827e5227f9cf084976eeb738b22c7eb7665bb5faa251bce41e09
+glue,1.4.1,f8b687d35cacb5ee7fcada6e9c26ea20c04d0bdc9d37e919a03abd1137513bc8
+hms,0.5.3,4e2b67c8cf65fe86179f24f42d82b3ca9377d5907837bda98b4fc6c2318853ad
+lifecycle,0.2.0,29746e8dee05d4e36f9c612e8c7a903a4f648a36b3b94c9776e518c38a412224
+magrittr,1.5,05c45943ada9443134caa0ab24db4a962b629f00b755ccf039a2a2a7b2c92ae8
+pillar,1.4.4,e1eaba75a937e90904334c6acc525e184d584e3601ac50c78b27cf1caeab3e6e
+pkgconfig,2.0.3,330fef440ffeb842a7dcfffc8303743f1feae83e8d6131078b5a44ff11bc3850
+plyr,1.8.6,ea55d26f155443e9774769531daa5d4c20a0697bb53abd832e891b126c935287
+readr,1.3.1,33f94de39bb7f2a342fbb2bd4e5afcfec08798eac39672ee18042ac0b349e4f3
+rlang,0.4.6,3a81b107765fd6ac0ad716c428d01878775ded9208ba125d43c890c73d2533ca
+tibble,3.0.1,154552cfb767e0bd48ef3fc61df8286ce52205e970815c5bc2560f41eceea79e
+utf8,1.1.4,f6da9cadfc683057d45f54b43312a359cf96ec2731c0dda18a8eae31d1e31e54
+vctrs,0.3.1,17e6358735504166ecb1aab581e5fa5e565ffb6f10e8a12c4d476a8e1f8aba08
```

Comment:
you have to add transitive deps, huh? bummer.

Reply:
=(